samples/web/content/testrtc/js/call.js

```diff
@@ -48,15 +48,19 @@ Call.prototype = {
     this.constrainOfferToRemoveVideoFec_ = true;
   },
 
-  // When the peerConnection is closed the callback is called once returning
-  // with an array of gathered stats.
-  gatherStats: function(peerConnection, callback, interval) {
+  // When the peerConnection is closed the statsFinishedCallback is called once
+  // returning with an array of gathered stats. timeForFirstFrameCallback will
+  // be called once when googFrameRateInput is > 0 with a timestamp in
+  // milliseconds.
+  // TODO (jansson) expand to cover audio as well.
+  gatherStats: function(peerConnection, statsCb, firstFrameCb, interval) {
+    var firstFrameCbSent = false;
     var stats = [];
     getStats_();
 
     function getStats_() {
       if (peerConnection.signalingState === 'closed') {
-        callback(stats);
+        statsCb(stats);
```

Comment:
I shortened the callback names due to grunt (jshint and JSCS) cannot handle function definitions with linebreaks.